sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -883,6 +883,129 @@ class Dataset[T] private[sql](
     println(showString(numRows, truncate, vertical))
   // scalastyle:on println
 
+  /**
+   * Gets top 20 rows of Dataset as string in a tabular form. Strings more than 20 characters
+   * will be truncated, and all cells will be aligned right.
+   *
+   * @group action
+   * @since 3.5.0
```

Comment:
nit: Why is this `3.5.0`?

Reply:
Thanks for reviewing @jaceklaskowski .  I took the version from one of the recent bugs but i realised i took the affected version instead of fix version which is wrong. I am trying to get details on how to get current release version to add but not able to find any links i can be confident on .